client/src/App.js

```diff
@@ -16,28 +16,28 @@ function App() {
       <GlobalStyle />
       <Switch>
         <Route exact path="/">
-          <CategoriesPage />
+          <CategoriesPage title={"Home"} />
         </Route>
         <Route path="/loading">
           <LoadingPage />
         </Route>
         <Route path="/product-overview">
-          <ProductsOverviewPage />
+          <ProductsOverviewPage title={"Angebote"} />
         </Route>
         <Route path="/product-detail">
-          <ProductPage />
+          <ProductPage title={"Angebot"} />
```

Comment:
the titles are pretty similar here- maybe change it to "Angebotsübersicht" ?

Reply:
Good point. I'm not sure how to finally name them in the end right now ...